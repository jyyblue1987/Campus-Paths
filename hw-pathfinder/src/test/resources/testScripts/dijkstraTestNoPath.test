# Find a path with Dijkstra's algorithm when the path is empty

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A C 1.0

FindPath simpleGraph A B