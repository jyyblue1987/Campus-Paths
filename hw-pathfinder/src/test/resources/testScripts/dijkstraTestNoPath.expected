# Find a path with Dijkstra's algorithm when the path is empty

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 1.000 from A to C in simpleGraph

path from A to B:
no path found